Source/WebCore/platform/mediastream/mac/CoreAudioSharedUnit.h

```diff
@@ -128,11 +127,11 @@ class CoreAudioSharedUnit final : public BaseAudioSharedUnit {
     // Only read/modified from the IO thread.
     Vector<Ref<AudioSampleDataSource>> m_activeSources;
 
-    CAAudioStreamDescription m_microphoneProcFormat;
+    std::optional<CAAudioStreamDescription> m_microphoneProcFormat;
```

Comment:
Ditto here, we might be able to remove m_microphoneProcFormat and use m_microphoneSampleBuffer->m_internalFormat.
Could be done as a follow-up.